ckanext/scheming/presets.json

```diff
@@ -113,9 +113,9 @@
       }
     },
     {
-      "preset_name": "json",
+      "preset_name": "json_object",
       "values": {
-        "validators": "ignore_missing scheming_valid_json",
+        "validators": "ignore_missing scheming_valid_json_object",
```

Comment:
use scheming_required instead of ignore_missing for the required setting to work on fields using this preset